openedx/features/edly/tests/test_organizations.py

```diff
@@ -0,0 +1,76 @@
+"""Tests covering the Organizations listing on the Studio home."""
+import logging
+
+from django.conf import settings
+from django.test import TestCase
+from django.urls import reverse
+from mock import patch
+from testfixtures import LogCapture
+from waffle.testutils import override_switch
+
+from openedx.features.edly.tests.factories import (EdlyOrganizationFactory,
```

Comment:
Need to fix it according to previous feedback here: https://github.com/edly-io/edx-platform/pull/72#discussion_r421987299